server/odc-core/src/main/java/com/oceanbase/odc/core/session/InMemoryConnectionSessionRepository.java

```diff
@@ -43,6 +43,9 @@ public class InMemorySessionRepository implements ConnectionSessionRepository {
     @Override
     public String store(@NonNull ConnectionSession session) {
         Validate.notNull(session.getId(), "Session.Id can not be null");
+        if (this.sessionId2SessionMap.containsKey(session.getId())) {
```

Comment:
So what will happen if generate a duplicated sessionId?

Reply:
as you can see, an exception will be thrown and the user will be prompted that failed to enter a database. he/she need to re-entry the database. 

> Will the session be destroied?

no, ODC will do nothing.